README.md

```diff
@@ -1,122 +1,28 @@
-on multiple software projects concurrently may find this helpful to avoid port conflicts.
-
-To do so, before running any `docker-compose` commands, set any of the environment variables:
-* `DOCKER_POSTGRES_PORT`
-* `DOCKER_RABBITMQ_PORT`
-* `DOCKER_MINIO_PORT`
-
-The Django server must be informed about the changes:
-* When running the "Develop with Docker" configuration, override the environment variables:
-  * `DJANGO_MINIO_STORAGE_MEDIA_URL`, using the port from `DOCKER_MINIO_PORT`.
-* When running the "Develop Natively" configuration, override the environment variables:
-  * `DJANGO_DATABASE_URL`, using the port from `DOCKER_POSTGRES_PORT`
-  * `DJANGO_CELERY_BROKER_URL`, using the port from `DOCKER_RABBITMQ_PORT`
-  * `DJANGO_MINIO_STORAGE_ENDPOINT`, using the port from `DOCKER_MINIO_PORT`
-
-Since most of Django's environment variables contain additional content, use the values from
-the appropriate `dev/.env.docker-compose*` file as a baseline for overrides.
-
-## Testing
-### Initial Setup
-tox is used to execute all tests.
-tox is installed automatically with the `dev` package extra.
-To install the tox pytest dependencies into your environment, run `pip install -e .[test]`.
-These are useful for IDE autocompletion or if you want to run `pytest` directly (not recommended).
-
-When running the "Develop with Docker" configuration, all tox commands must be run as
-`docker-compose run --rm django tox`; extra arguments may also be appended to this form.
-
-### Running Tests
-Run `tox` to launch the full test suite.
-
-Individual test environments may be selectively run.
-This also allows additional options to be be added.
-Useful sub-commands include:
-* `tox -e lint`: Run only the style checks
-* `tox -e type`: Run only the type checks
-* `tox -e test`: Run only the pytest-driven tests
-* `tox -e test -- -k test_file`: Run the pytest-driven tests that match the regular expression `test_file`
-* `tox -e test -- --cov=dandiapi`: Generate a test coverage report
-* `tox -e test -- --cov=dandiapi --cov-report=html`: Generate an HTML test coverage report in the `htmlcov` directory
-
-To automatically reformat all code to comply with
-some (but not all) of the style checks, run `tox -e format`.
-
-## API Authentication
-Read-only API endpoints (i.e. `GET`, `HEAD`) do not require any
-authentication. All other endpoints require token authentication
-to call.
-
-### Creating a Token
-Visit the URL `/admin` with a web browser, logging
-in with the credentials entered during the `createsuperuser` setup step..
-Then go to `/swagger` and use `GET /auth/token` end-point.
-
-### Supplying the Token
-In API endpoint calls, add the `Authorization` HTTP header with a value of
-`Token <token_value>`. For example, for a token `1234`, include the header:
-`Authorization: Token 1234`.
-
-## Custom management scripts
-Documentation for the various custom `manage.py` scripts can be found in `DEVELOPMENT.md`.
+To hack on the archive codebase:
```

Comment:
```suggestion
To hack on the archive django codebase:
```

Reply:
I went a step further with the change in f923cf4; let me know what you think.